packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/new/new.html

```diff
@@ -2,27 +2,43 @@
     data-ovh-alert="app.dedicatedClouds.datacenter.backup.new"
     id="backupNewMainAlert"
 ></div>
-<p data-translate="dedicatedCloud_datacenter_backup_new_description1"></p>
-<p
-    data-ng-bind-html="::'dedicatedCloud_datacenter_backup_new_description2' | translate"
-></p>
-<strong>
-    <a
-        target="_blank"
-        rel="noopener"
-        class="oui-link_icon"
-        data-ng-href="{{ ::$ctrl.VEEAM_TARIFF_DETAILS_URL }}"
-    >
-        <span
-            data-translate="dedicatedCloud_datacenter_backup_new_tariff_url"
-        ></span>
-        <span
-            class="oui-icon oui-icon-arrow-right_extra-thin"
-            aria-hidden="true"
-        ></span>
-    </a>
-</strong>
-<oui-stepper class="mt-3" data-on-finish="$ctrl.orderBackup()">
+<div data-ng-if="!$ctrl.backup.isLegacy()">
+    <p data-translate="dedicatedCloud_datacenter_backup_new_description1"></p>
+    <p
+        data-ng-bind-html="::'dedicatedCloud_datacenter_backup_new_description2' | translate"
+    ></p>
+    <strong>
+        <a
+            target="_blank"
+            rel="noopener"
+            class="oui-link_icon"
+            data-ng-href="{{ ::$ctrl.VEEAM_TARIFF_DETAILS_URL }}"
+        >
+            <span
+                data-translate="dedicatedCloud_datacenter_backup_new_tariff_url"
+            ></span>
+            <span
+                class="oui-icon oui-icon-arrow-right_extra-thin"
+                aria-hidden="true"
+            ></span>
+        </a>
+    </strong>
+</div>
```

Comment:
I would have done the opposite

```suggestion
        <a
            target="_blank"
            rel="noopener"
            class="oui-link_icon"
            data-ng-href="{{ ::$ctrl.VEEAM_TARIFF_DETAILS_URL }}"
        >
           <strong>
               <span
                    data-translate="dedicatedCloud_datacenter_backup_new_tariff_url"
                ></span>
            </strong>
            <span
                class="oui-icon oui-icon-arrow-right_extra-thin"
                aria-hidden="true"
            ></span>
        </a>
</div>
```